src/js/60-docs-roles.js

```diff
@@ -63,12 +65,16 @@ document.addEventListener('DOMContentLoaded', function () {
   const roleDivs = document.querySelectorAll('body.docs:not(.docshome) *[class*="label--"]')
 
   roleDivs.forEach(function (roleDiv) {
+    var roles = roleDiv.classList
+    //  ignore discrete headers
```

Comment:
I'd move the comment to before line 76, where the check happens.

Reply:
Argh - well spotted. I actually meant to combine the if statements before committing